src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -175,6 +181,7 @@ def _alloc_node(self, node, requested_cores, requested_gpus,
         requested (but the call will never return more than requested).
         '''
 
+        # MING: Maybe we can change the function name to _find_alloc_in_node ?
```

Comment:
Maybe :)

Reply:
the function is now called `find_resources`, and I'm ok with that.